sa/_db/migrations/20170423163751_CreateAuthorizationsTable.sql

```diff
@@ -0,0 +1,30 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+CREATE TABLE `authorizations` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `token` varchar(255) NOT NULL,
+  `identifierValue` varchar(255) DEFAULT NULL,
```

Comment:
We try to use `NOT NULL` whenever possible. I think all of the rows below can be set to `NOT NULL`.